os/arch/arm/src/amebad/Kconfig

```diff
@@ -73,6 +73,10 @@ config FTL_ENABLED
 	bool "Enable FTL"
 	default n
 
+config BLE_DEBUG
```

Comment:
How about adding prefix as `AMEBAD`?
Because
1. All of others have prefix.
2. When we see defconfig, it could cause misunderstanding like it's common module config.

Reply:
Your are right, i added AMEBAD prefix.

Reply:
It's not added. Could you check again?

Reply:
Sorry, my mistake, the commit did not add the changes file.
It is added now.